components/04-reference-pages/07-faculty-profile-page/faculty-profile-page.hbs

```diff
@@ -54,98 +56,140 @@
+                            target="_blank">2017 - Go Code Colorado Finalist, State of Colorado</a></li>
+                        <li><a href="https://www.csupueblo.edu/news/2017/04-20-outstanding-faculty-staff-luncheon.html"
+                            target="_parent">CSU-Pueblo honors outstanding faculty and staff at luncheon</a></li>
+                        <li><a
+                            href="https://www.csupueblo.edu/classified-staff-council/employee-of-the-year-awards.html"
+                            target="_parent">Employee of the Year Awards</a></li>
+                        <li><a href="https://socopressclub.org/awards/media-excellence/" target="_parent">Media
+                            Excellence</a></li>
+                      </ul>
+                    </div>
+                  </div>
+                </div>
+
+                <div id="results"></div>
+              </div>
+            </div>
           </div>
         </div>
       </div>
     </section>
 
-    <section class="faculty-contact-sec">
+    <!--BEGIN UTSA DISCOVERY CODE-->
+    <section class="position-relative py-5">
       <div class="container">
-        <div class="faculty-contact-header">
-          <h5>About</h5>
-          <p>Dr. Juan Manuel Sanchez obtained his Ph.D. in accounting from The University of Texas at San Antonio in
-            2006 after working for global firms including Deloitte (a global accounting firm) and SBC Communications
-            (now AT&amp;T). Dr. Sanchez has published in elite accounting, finance and multidisciplinary journals such
-            as The Accounting Review, Journal of Accounting Research, Journal of Financial Economics, Journal of
-            Business Ethics, Journal of Business Finance and Accounting, Accounting Horizons, Journal of Accounting and
-            Public Policy, Journal of Accounting, Auditing and Finance, and Journal of Information Systems.</p>
-          <p>His research is widely cited by academics and major media outlets including, The Wall Street Journal,
-            CFO.com, Accounting Today, Compliance Week, and Harvard Law School Forum on Corporate Governance and
-            Financial Regulation. According to BYU Author Rankings Sanchez’ research ranks him in the first (top) decile
-            among all accounting researchers. For more details about his research, please visit his SSRN Author Page.
-          </p>
-        </div>
-        <div class="faculty-research-interest">
-          <h5>Research Interests</h5>
-          <ul>
-            <li>Executive compensation</li>
-            <li>Corporate governance</li>
-            <li>Financial reporting issues</li>
-            <li>Accounting information systems</li>
-          </ul>
-        </div>
-      </div>
-    </section>
+        <div class="profile-details-component-div white">
+          <div class="col-xxl-12 col-xl-12 col-lg-12 col-md-12 col-sm-12" id="results">
```

Comment:
**div class="col-12"**  is all that is necessary here.

Reply:
k, think I got them all =)